pkg/cmd/stack/translate.go

```diff
@@ -43,7 +43,7 @@ func translate(s *model.Stack, forceBuild, noCache bool) error {
 		return nil
 	}
 
-	c, _, configNamespace, err := k8Client.GetLocal()
+	c, _, configNamespace, err := k8Client.GetLocal("")
```

Comment:
why is it empty here? if we are considering the context, don't we need it in all the k8s calls?